### PR TITLE
OE-827: Add back grid and update top checkout lanes URL

### PR DESCRIPTION
**Pull request by Toxiapo**
This PR adds the grid section and updates the lane. 

I also moved a couple of components to a `Homepage` folder because the `pages/index` file for the Homepage was getting clumsy and hard to read.

### CODE REVIEW

File: apps/oew/src/components/chakra-ui/HomePage/GridSection.tsx [hunk 1]
```diff
@@ -2,12 +2,14 @@ import { SimpleGrid, Box, Text } from "@chakra-ui/react";
...
-}: {
+type GridSection = {
   gridBooks: SimpleBook[];
-}) {
+  title: string;
+};
+
+const GridSection: React.FC<GridSection> = ({ gridBooks, title }) => {
   if (!gridBooks.length) return null;
+  console.log(gridBooks);
```
Comment (oliviawongnyc) [inline apps/oew/src/components/chakra-ui/HomePage/GridSection.tsx:12]: Did you intend to keep this?

**Review by oliviawongnyc — APPROVED** [review]
This looks great! Nice cleaning up of the files and fetch calls. I am still shocked and confused about this whole fetching a hidden lane thing, but also am happy it is working! Will definitely have to revisit the Are You Lost lane.

**Comment by samanthaandrews** [comment]
Very much agree with @oliviawongnyc that this cleanup is awesome! And I second her approval. Let's get this over to @jotten for visual QA (if you haven't already).

**Review by kristojorg — APPROVED** [review]
Looks good!

File: apps/oew/src/components/chakra-ui/HomePage/GridSection.tsx (hunk 1 above)
Comment (Toxiapo) [reply apps/oew/src/components/chakra-ui/HomePage/GridSection.tsx]: updated, ty!

**Comment by samanthaandrews** [comment]
Is this still pending QA? Or maybe it's being reviewed by Miguel?

**Comment by Toxiapo** [comment]
@samanthaandrews It's done! I will merge it.

**Pull request merged**